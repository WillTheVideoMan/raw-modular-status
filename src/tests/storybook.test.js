import initStoryshots from "@storybook/addon-storyshots";
Date.now = jest.fn(() => 965931720000);
initStoryshots();
