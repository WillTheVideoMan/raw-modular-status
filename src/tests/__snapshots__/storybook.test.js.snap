// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Composite/CardList Default 1`] = `
<div
  style={
    Object {
      "marginTop": "1rem",
    }
  }
>
  <div>
    <div
      className="sc-VigVT hgpqRh"
      style={
        Object {
          "borderWidth": "0rem",
          "height": "8.3rem",
        }
      }
    />
    <div>
      <div
        style={
          Object {
            "height": "0rem",
            "opacity": 0,
            "transform": "translateX(8rem)",
          }
        }
      >
        <div
          className="sc-EHOje byzvOr"
          style={
            Object {
              "height": "8rem",
            }
          }
        >
          <div
            className="sc-bZQynM jzLdNW"
            style={
              Object {
                "width": "8rem",
              }
            }
          >
            <div
              className="sc-dnqmqq kqTqqP"
              style={
                Object {
                  "fontSize": "4rem",
                }
              }
            >
              12X
            </div>
          </div>
          <div
            className="sc-htoDjs dRcTZd"
            style={
              Object {
                "lineHeight": "1.25",
              }
            }
          >
            <div
              className="sc-iwsKbI jzKbeF"
            >
              20:12
            </div>
            <div
              className="sc-gZMcBi ceWbWy"
            >
              Towards Leamington
            </div>
          </div>
        </div>
      </div>
      <div
        style={
          Object {
            "height": "0rem",
            "opacity": 0,
            "transform": "translateX(8rem)",
          }
        }
      >
        <div
          className="sc-EHOje byzvOr"
          style={
            Object {
              "height": "8rem",
            }
          }
        >
          <div
            className="sc-bZQynM jzLdNW"
            style={
              Object {
                "width": "8rem",
              }
            }
          >
            <img
              className="sc-gzVnrw dUtZaE"
              src="https://media.radio.warwick.ac.uk/shows/5010.large.jpg"
            />
          </div>
          <div
            className="sc-htoDjs dRcTZd"
            style={
              Object {
                "lineHeight": "1.25",
              }
            }
          >
            <div
              className="sc-iwsKbI jzKbeF"
            >
              Psychademics
            </div>
            <div
              className="sc-gZMcBi ceWbWy"
            >
              20:00 - 21:00
            </div>
          </div>
        </div>
      </div>
      <div
        style={
          Object {
            "height": "0rem",
            "opacity": 0,
            "transform": "translateX(8rem)",
          }
        }
      >
        <div
          className="sc-EHOje byzvOr"
          style={
            Object {
              "height": "8rem",
            }
          }
        >
          <div
            className="sc-bZQynM jzLdNW"
            style={
              Object {
                "width": "8rem",
              }
            }
          >
            <img
              className="sc-gzVnrw dUtZaE"
              src="https://images-na.ssl-images-amazon.com/images/I/A1gZc70vUIL._SL1500_.jpg"
            />
          </div>
          <div
            className="sc-htoDjs dRcTZd"
            style={
              Object {
                "lineHeight": "1.25",
              }
            }
          >
            <div
              className="sc-iwsKbI jzKbeF"
            >
              Feels Like We Only Go Backwards (Special Edition)
            </div>
            <div
              className="sc-gZMcBi ceWbWy"
            >
              Tame Impala
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Composite/CardList Empty 1`] = `
<div
  style={
    Object {
      "marginTop": "1rem",
    }
  }
>
  <div>
    <div
      className="sc-VigVT hgpqRh"
      style={
        Object {
          "borderWidth": "0rem",
          "height": "8.3rem",
        }
      }
    >
      <div
        className="sc-gqjmRU gifDXn"
      >
        Nothing To Show
      </div>
    </div>
    <div />
  </div>
</div>
`;

exports[`Storyshots Composite/CardList Highlighted 1`] = `
<div
  style={
    Object {
      "marginTop": "1rem",
    }
  }
>
  <div>
    <div
      className="sc-VigVT hgpqRh"
      style={
        Object {
          "borderWidth": "0.2rem",
          "height": "8.3rem",
        }
      }
    />
    <div>
      <div
        style={
          Object {
            "height": "0rem",
            "opacity": 0,
            "transform": "translateX(8rem)",
          }
        }
      >
        <div
          className="sc-EHOje byzvOr"
          style={
            Object {
              "height": "8rem",
            }
          }
        >
          <div
            className="sc-bZQynM jzLdNW"
            style={
              Object {
                "width": "8rem",
              }
            }
          >
            <div
              className="sc-dnqmqq kqTqqP"
              style={
                Object {
                  "fontSize": "4rem",
                }
              }
            >
              12X
            </div>
          </div>
          <div
            className="sc-htoDjs dRcTZd"
            style={
              Object {
                "lineHeight": "1.25",
              }
            }
          >
            <div
              className="sc-iwsKbI jzKbeF"
            >
              11:31
            </div>
            <div
              className="sc-gZMcBi ceWbWy"
            >
              Towards Coventry
            </div>
          </div>
        </div>
      </div>
      <div
        style={
          Object {
            "height": "0rem",
            "opacity": 0,
            "transform": "translateX(8rem)",
          }
        }
      >
        <div
          className="sc-EHOje byzvOr"
          style={
            Object {
              "height": "8rem",
            }
          }
        >
          <div
            className="sc-bZQynM jzLdNW"
            style={
              Object {
                "width": "8rem",
              }
            }
          >
            <img
              className="sc-gzVnrw dUtZaE"
              src="https://media.radio.warwick.ac.uk/shows/5047.large.jpg"
            />
          </div>
          <div
            className="sc-htoDjs dRcTZd"
            style={
              Object {
                "lineHeight": "1.25",
              }
            }
          >
            <div
              className="sc-iwsKbI jzKbeF"
            >
              The Acoustic Lounge
            </div>
            <div
              className="sc-gZMcBi ceWbWy"
            >
              11:00 - 12:00
            </div>
          </div>
        </div>
      </div>
      <div
        style={
          Object {
            "height": "0rem",
            "opacity": 0,
            "transform": "translateX(8rem)",
          }
        }
      >
        <div
          className="sc-EHOje byzvOr"
          style={
            Object {
              "height": "8rem",
            }
          }
        >
          <div
            className="sc-bZQynM jzLdNW"
            style={
              Object {
                "width": "8rem",
              }
            }
          >
            <img
              className="sc-gzVnrw dUtZaE"
              src="https://images-na.ssl-images-amazon.com/images/I/A1LVEJikmZL._SL1500_.jpg"
            />
          </div>
          <div
            className="sc-htoDjs dRcTZd"
            style={
              Object {
                "lineHeight": "1.25",
              }
            }
          >
            <div
              className="sc-iwsKbI jzKbeF"
            >
              New Person (Same Old Mistakes)
            </div>
            <div
              className="sc-gZMcBi ceWbWy"
            >
              Tame Impala
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Composite/CardList Mixed 1`] = `
<div
  style={
    Object {
      "marginTop": "1rem",
    }
  }
>
  <div>
    <div
      className="sc-VigVT hgpqRh"
      style={
        Object {
          "borderWidth": "0rem",
          "height": "8.3rem",
        }
      }
    />
    <div>
      <div
        style={
          Object {
            "height": "0rem",
            "opacity": 0,
            "transform": "translateX(8rem)",
          }
        }
      >
        <div
          className="sc-EHOje byzvOr"
          style={
            Object {
              "height": "8rem",
            }
          }
        >
          <div
            className="sc-bZQynM jzLdNW"
            style={
              Object {
                "width": "8rem",
              }
            }
          >
            <div
              className="sc-dnqmqq kqTqqP"
              style={
                Object {
                  "fontSize": "4rem",
                }
              }
            >
              12X
            </div>
          </div>
          <div
            className="sc-htoDjs dRcTZd"
            style={
              Object {
                "lineHeight": "1.25",
              }
            }
          >
            <div
              className="sc-iwsKbI jzKbeF"
            >
              17:02
            </div>
            <div
              className="sc-gZMcBi ceWbWy"
            >
              Towards Leamington
            </div>
          </div>
        </div>
      </div>
      <div
        style={
          Object {
            "height": "0rem",
            "opacity": 0,
            "transform": "translateX(8rem)",
          }
        }
      >
        <div
          className="sc-EHOje byzvOr"
          style={
            Object {
              "height": "6rem",
            }
          }
        >
          <div
            className="sc-bZQynM jzLdNW"
            style={
              Object {
                "width": "6rem",
              }
            }
          >
            <img
              className="sc-gzVnrw dUtZaE"
              src="https://media.radio.warwick.ac.uk/shows/5100.large.jpg"
            />
          </div>
          <div
            className="sc-htoDjs dRcTZd"
            style={
              Object {
                "lineHeight": "1",
              }
            }
          >
            <div
              className="sc-iwsKbI jzKbeF"
            >
              The Italian Football Show
            </div>
            <div
              className="sc-gZMcBi ceWbWy"
            >
              17:00 - 18:00
            </div>
          </div>
        </div>
      </div>
      <div
        style={
          Object {
            "height": "0rem",
            "opacity": 0,
            "transform": "translateX(8rem)",
          }
        }
      >
        <div
          className="sc-EHOje byzvOr"
          style={
            Object {
              "height": "6rem",
            }
          }
        >
          <div
            className="sc-bZQynM jzLdNW"
            style={
              Object {
                "width": "6rem",
              }
            }
          >
            <img
              className="sc-gzVnrw dUtZaE"
              src="https://i.ytimg.com/vi/hNJOI2dtDZ4/maxresdefault.jpg"
            />
          </div>
          <div
            className="sc-htoDjs dRcTZd"
            style={
              Object {
                "lineHeight": "1",
              }
            }
          >
            <div
              className="sc-iwsKbI jzKbeF"
            >
              Borderline
            </div>
            <div
              className="sc-gZMcBi ceWbWy"
            >
              Tame Impala
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Composite/CardList Slim 1`] = `
<div
  style={
    Object {
      "marginTop": "1rem",
    }
  }
>
  <div>
    <div
      className="sc-VigVT hgpqRh"
      style={
        Object {
          "borderWidth": "0rem",
          "height": "6.3rem",
        }
      }
    />
    <div>
      <div
        style={
          Object {
            "height": "0rem",
            "opacity": 0,
            "transform": "translateX(8rem)",
          }
        }
      >
        <div
          className="sc-EHOje byzvOr"
          style={
            Object {
              "height": "6rem",
            }
          }
        >
          <div
            className="sc-bZQynM jzLdNW"
            style={
              Object {
                "width": "6rem",
              }
            }
          >
            <div
              className="sc-dnqmqq kqTqqP"
              style={
                Object {
                  "fontSize": "3rem",
                }
              }
            >
              U1
            </div>
          </div>
          <div
            className="sc-htoDjs dRcTZd"
            style={
              Object {
                "lineHeight": "1",
              }
            }
          >
            <div
              className="sc-iwsKbI jzKbeF"
            >
              19:25
            </div>
            <div
              className="sc-gZMcBi ceWbWy"
            >
              Towards Coventry
            </div>
          </div>
        </div>
      </div>
      <div
        style={
          Object {
            "height": "0rem",
            "opacity": 0,
            "transform": "translateX(8rem)",
          }
        }
      >
        <div
          className="sc-EHOje byzvOr"
          style={
            Object {
              "height": "6rem",
            }
          }
        >
          <div
            className="sc-bZQynM jzLdNW"
            style={
              Object {
                "width": "6rem",
              }
            }
          >
            <img
              className="sc-gzVnrw dUtZaE"
              src="https://media.radio.warwick.ac.uk/shows/5163.large.jpg"
            />
          </div>
          <div
            className="sc-htoDjs dRcTZd"
            style={
              Object {
                "lineHeight": "1",
              }
            }
          >
            <div
              className="sc-iwsKbI jzKbeF"
            >
              ReggaetÓN
            </div>
            <div
              className="sc-gZMcBi ceWbWy"
            >
              19:00 - 20:00
            </div>
          </div>
        </div>
      </div>
      <div
        style={
          Object {
            "height": "0rem",
            "opacity": 0,
            "transform": "translateX(8rem)",
          }
        }
      >
        <div
          className="sc-EHOje byzvOr"
          style={
            Object {
              "height": "6rem",
            }
          }
        >
          <div
            className="sc-bZQynM jzLdNW"
            style={
              Object {
                "width": "6rem",
              }
            }
          >
            <img
              className="sc-gzVnrw dUtZaE"
              src="http://www.brooklynvegan.com/files/2019/03/tame-impala-patience.jpg"
            />
          </div>
          <div
            className="sc-htoDjs dRcTZd"
            style={
              Object {
                "lineHeight": "1",
              }
            }
          >
            <div
              className="sc-iwsKbI jzKbeF"
            >
              Patience
            </div>
            <div
              className="sc-gZMcBi ceWbWy"
            >
              Tame Impala
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Simple/Button Default 1`] = `
<div
  className="sc-htpNat hriTYS"
  disabled={false}
  onClick={[Function]}
  onMouseDown={[Function]}
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchStart={[Function]}
  style={
    Object {
      "boxShadow": "0px 0px 0px 0px rgba(0, 0, 0, 0)",
      "transform": "translateY(0px)",
    }
  }
>
  <div>
    <div
      className="sc-bxivhb hJcVJR"
    >
      Click Me!
    </div>
  </div>
</div>
`;

exports[`Storyshots Simple/Button Disabled 1`] = `
<div
  className="sc-htpNat gquIDs"
  disabled={true}
  onClick={null}
  onMouseDown={[Function]}
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchStart={[Function]}
  style={
    Object {
      "boxShadow": "0px 0px 0px 0px rgba(0, 0, 0, 0)",
      "transform": "translateY(0px)",
    }
  }
>
  <div>
    <div
      className="sc-bxivhb hJcVJR"
    >
      You Can't Click Me :(
    </div>
  </div>
</div>
`;

exports[`Storyshots Simple/Button Loading 1`] = `
<div
  className="sc-htpNat hriTYS"
  onClick={null}
  onMouseDown={[Function]}
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchStart={[Function]}
  style={
    Object {
      "boxShadow": "0px 0px 0px 0px rgba(0, 0, 0, 0)",
      "transform": "translateY(0px)",
    }
  }
>
  <div
    className="sc-bdVaJa bNlOCv"
  >
    <div
      className="sc-bwzfXH ghagIV"
      size={0.5}
    />
    <div
      className="sc-bwzfXH bvVunS"
      size={0.5}
    />
    <div
      className="sc-bwzfXH kZQvcK"
      size={0.5}
    />
  </div>
</div>
`;

exports[`Storyshots Simple/Button With Icon & Text 1`] = `
<div
  className="sc-htpNat hriTYS"
  onClick={[Function]}
  onMouseDown={[Function]}
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchStart={[Function]}
  style={
    Object {
      "boxShadow": "0px 0px 0px 0px rgba(0, 0, 0, 0)",
      "transform": "translateY(0px)",
    }
  }
>
  <div>
    <div
      className="sc-bxivhb hJcVJR"
    >
      Click Me!
    </div>
    <i
      className="sc-ifAKCX eQYFnI"
    >
      fingerprint
    </i>
  </div>
</div>
`;

exports[`Storyshots Simple/Button With Icon 1`] = `
<div
  className="sc-htpNat hriTYS"
  onClick={[Function]}
  onMouseDown={[Function]}
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchStart={[Function]}
  style={
    Object {
      "boxShadow": "0px 0px 0px 0px rgba(0, 0, 0, 0)",
      "transform": "translateY(0px)",
    }
  }
>
  <div>
    <i
      className="sc-ifAKCX eQYFnI"
    >
      fingerprint
    </i>
  </div>
</div>
`;

exports[`Storyshots Simple/Card Default 1`] = `
<div
  style={
    Object {
      "marginTop": "1rem",
    }
  }
>
  <div
    className="sc-EHOje byzvOr"
    style={
      Object {
        "height": "8rem",
      }
    }
  >
    <div
      className="sc-bZQynM jzLdNW"
      style={
        Object {
          "width": "8rem",
        }
      }
    >
      <div
        className="sc-dnqmqq kqTqqP"
        style={
          Object {
            "fontSize": "4rem",
          }
        }
      >
        12X
      </div>
    </div>
    <div
      className="sc-htoDjs dRcTZd"
      style={
        Object {
          "lineHeight": "1.25",
        }
      }
    >
      <div
        className="sc-iwsKbI jzKbeF"
      >
        20:12
      </div>
      <div
        className="sc-gZMcBi ceWbWy"
      >
        Towards leamington
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Simple/Card No Hero 1`] = `
<div
  style={
    Object {
      "marginTop": "1rem",
    }
  }
>
  <div
    className="sc-EHOje byzvOr"
    style={
      Object {
        "height": "8rem",
      }
    }
  >
    <div
      className="sc-htoDjs dRcTZd"
      style={
        Object {
          "lineHeight": "1.25",
        }
      }
    >
      <div
        className="sc-iwsKbI jzKbeF"
      >
        Title
      </div>
      <div
        className="sc-gZMcBi ceWbWy"
      >
        Subtitle
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Simple/Card Slim 1`] = `
<div
  style={
    Object {
      "marginTop": "1rem",
    }
  }
>
  <div
    className="sc-EHOje byzvOr"
    style={
      Object {
        "height": "6rem",
      }
    }
  >
    <div
      className="sc-bZQynM jzLdNW"
      style={
        Object {
          "width": "6rem",
        }
      }
    >
      <img
        className="sc-gzVnrw dUtZaE"
        src="https://media.radio.warwick.ac.uk/shows/5010.large.jpg"
      />
    </div>
    <div
      className="sc-htoDjs dRcTZd"
      style={
        Object {
          "lineHeight": "1",
        }
      }
    >
      <div
        className="sc-iwsKbI jzKbeF"
      >
        Psychademics
      </div>
      <div
        className="sc-gZMcBi ceWbWy"
      >
        20:00 - 21:00
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Simple/Card With Image 1`] = `
<div
  style={
    Object {
      "marginTop": "1rem",
    }
  }
>
  <div
    className="sc-EHOje byzvOr"
    style={
      Object {
        "height": "8rem",
      }
    }
  >
    <div
      className="sc-bZQynM jzLdNW"
      style={
        Object {
          "width": "8rem",
        }
      }
    >
      <img
        className="sc-gzVnrw dUtZaE"
        src="https://media.radio.warwick.ac.uk/shows/5010.large.jpg"
      />
    </div>
    <div
      className="sc-htoDjs dRcTZd"
      style={
        Object {
          "lineHeight": "1.25",
        }
      }
    >
      <div
        className="sc-iwsKbI jzKbeF"
      >
        Psychademics
      </div>
      <div
        className="sc-gZMcBi ceWbWy"
      >
        20:00 - 21:00
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Simple/Card With Long Text 1`] = `
<div
  style={
    Object {
      "marginTop": "1rem",
    }
  }
>
  <div
    className="sc-EHOje byzvOr"
    style={
      Object {
        "height": "8rem",
      }
    }
  >
    <div
      className="sc-bZQynM jzLdNW"
      style={
        Object {
          "width": "8rem",
        }
      }
    >
      <img
        className="sc-gzVnrw dUtZaE"
        src="https://images-na.ssl-images-amazon.com/images/I/A1gZc70vUIL._SL1500_.jpg"
      />
    </div>
    <div
      className="sc-htoDjs dRcTZd"
      style={
        Object {
          "lineHeight": "1.25",
        }
      }
    >
      <div
        className="sc-iwsKbI jzKbeF"
      >
        Feels Like We Only Go Backwards (Special Edition)
      </div>
      <div
        className="sc-gZMcBi ceWbWy"
      >
        Tame Impala
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Simple/Headline Default 1`] = `
<div
  className="sc-fjdhpX bcSelW"
>
  Big Important Text
  <hr />
</div>
`;

exports[`Storyshots Simple/ImageCycle Default 1`] = `
<div
  className="sc-jzJRlG lkgTUP"
  style={
    Object {
      "backgroundImage": "url('https://dev.radio.warwick.ac.uk/status/images/main.jpg')",
      "opacity": 0,
      "transform": "scale(0.8)",
    }
  }
/>
`;

exports[`Storyshots Simple/ImageCycle No Animation 1`] = `
<div
  className="sc-jzJRlG lkgTUP"
  style={
    Object {
      "backgroundImage": "url('https://webcams.radio.warwick.ac.uk/st1-dj.jpg?force_fetch=965931720000')",
      "opacity": 1,
      "transform": "scale(1)",
    }
  }
/>
`;

exports[`Storyshots Simple/Loader Default 1`] = `
<div
  className="sc-bdVaJa bNlOCv"
>
  <div
    className="sc-bwzfXH bYnxHq"
    size={1}
  />
  <div
    className="sc-bwzfXH hnAMwj"
    size={1}
  />
  <div
    className="sc-bwzfXH iZsKlc"
    size={1}
  />
</div>
`;

exports[`Storyshots Simple/Message Default 1`] = `
<div
  style={
    Object {
      "marginTop": "1rem",
    }
  }
>
  <div
    className="sc-cSHVUG dxcUXa"
  >
    <div
      className="sc-kAzzGY fJVoOx"
    >
      <img
        className="sc-chPdSV dtIrRp"
        src="https://seeklogo.com/images/T/twitter-2012-negative-logo-5C6C1F1521-seeklogo.com.png"
      />
      <div
        className="sc-kgoBCf bjONhu"
      >
        <div
          className="sc-kGXeez ioiosA"
        >
          Student Radio Assoc.
        </div>
        <div
          className="sc-kpOJdX dstErn"
        >
          @SRA
        </div>
      </div>
    </div>
    <div
      className="sc-dxgOiQ cDGlIr"
    >
      @RAW1251AM have a Show Development Team.

They develop shows.

And get nominated for #ILSR awards 

🙌THIS https://t.co/CbsjmYzx39
    </div>
  </div>
</div>
`;

exports[`Storyshots Simple/Table Default 1`] = `
<table
  className="sc-ckVGcZ ipvkBA"
>
  <tbody>
    <tr>
      <th
        className="sc-jKJlTe bMeUVo"
      />
      <th
        className="sc-jKJlTe bMeUVo"
      >
        Studio 2
      </th>
      <th
        className="sc-jKJlTe bMeUVo"
      >
        Tascam
      </th>
    </tr>
    <tr>
      <td
        className="sc-eNQAEJ kOdNHk"
      >
        08:00
      </td>
      <td
        className="sc-eNQAEJ kOdNHk"
      >
        Will Hall
      </td>
      <td
        className="sc-eNQAEJ kOdNHk"
      >
        -
      </td>
    </tr>
    <tr>
      <td
        className="sc-eNQAEJ kOdNHk"
      >
        09:00
      </td>
      <td
        className="sc-eNQAEJ kOdNHk"
      >
        Barnany Merrill
      </td>
      <td
        className="sc-eNQAEJ kOdNHk"
      >
        -
      </td>
    </tr>
  </tbody>
</table>
`;

exports[`Storyshots Simple/TextInput Default 1`] = `
<div
  className="sc-hMqMXs kVEkNR"
>
  <input
    className="sc-kEYyzF eoOPFw"
    onBlur={[Function]}
    onChange={[Function]}
    onFocus={[Function]}
    placeholder="Type Here"
    type="text"
    value=""
  />
  <div
    className="sc-kkGfuU jinZKR"
    style={
      Object {
        "width": "0%",
      }
    }
  />
</div>
`;
